Adding clarity for candidates (#2)
appcues/recruiting-ui: README.md

- The application is deliberately written to be "not very clean code". It is **not** representative of good code at Appcues.
- This README is intentionally sparse, so you'll have to read the code to understand how the application works.

> - There are some things to avoid right from the get-go. 
>   - Don't spend the time to remove Redux from the project. If you believe an improvement can be made by removing it instead include it in the PR writeup. 
>   - We are less concerned about seeing things like code reorganization, styling improvements, or new features.

Reviewer: ```suggestion
- There are some things to avoid right from the get-go. 
  - Don't spend the time to remove Redux from the project. 
  - We are less concerned about seeing things like code reorganization, styling improvements, or new features.
  - If you believe an improvement can be made on any point above include it in the PR writeup.
```

suggestion only and open for discussion, IMO would be interesting to hear what the candidate thinks about any of the points not only about removing redux. Lemme know your thoughts!

Reviewer 2: I think this is a good suggestion

Author: Great change Diel, Thanks!
